src/OpenTelemetry/Metrics/MetricStreamConfiguration.cs

```diff
@@ -91,11 +95,39 @@ public class MetricStreamConfiguration
         }
     }
 
+    /// <summary>
+    /// Gets or sets a positive integer value
+    /// defining the maximum number of data points allowed to
+    /// per view.
+    /// </summary>
+    /// <remarks>
+    /// Note: If there is no matching view, but the MetricReader
```

Comment:
```suggestion
    /// Note: If there is no matching view, but the MeterProvider
```

I dont think we allow it per MetricReader, but this is rather applied to entire MeterProvider.. but please confirm..

Reply:
It gets passed from the provider to the reader I think that's where the confusion is coming from. I suggested changing the remarks wholly above.

Reply:
This is got mixed up with the spec requirement https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#configuration-1 😄

1. A [view](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view) with criteria matching the instrument an aggregation is created for has an aggregation_cardinality_limit value defined for the stream, that value SHOULD be used.
2. If there is no matching view, but the MetricReader defines a default cardinality limit value based on the instrument an aggregation is created for, that value SHOULD be used.
3. If none of the previous values are defined, the default value of 2000 SHOULD be used.